pkg/operator/csi/csidrivercontrollerservicecontroller/helpers.go

```diff
@@ -48,6 +49,32 @@ func WithObservedProxyDeploymentHook() dc.DeploymentHookFunc {
 	}
 }
 
+func WithTrustedCADeploymentHook(
+	configMapNamespace string,
+	configMapName string,
+	configMapInformer corev1.ConfigMapInformer,
+) dc.DeploymentHookFunc {
+	return func(_ *opv1.OperatorSpec, deployment *appsv1.Deployment) error {
+		cm, err := configMapInformer.Lister().ConfigMaps(configMapNamespace).Get(configMapName)
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		if err != nil {
+			return fmt.Errorf("failed to get ConfigMap %s/%s: %v", configMapNamespace, configMapName, err)
+		}
+		_, ok := cm.Data["ca-bundle.crt"]
+		if !ok {
+			return nil
+		}
+		containerNamesString := deployment.Annotations["config.openshift.io/inject-trusted-ca-bundle"]
```

Comment:
The Cluster Network Operator uses `(...)trusted-cabundle` (without `-`) to choose ConfigMaps to inject the custom bundle. Do I need to use the same?

Reply:
> Search other repos and be consistent. This would also be a reason to force the name of the config map you inject to be a name I don't prefer. I didn't think about this being the same as the CVO annotation.

Actually, I don't see this there.   I see this for configmaps: https://github.com/openshift/library-go/blob/d48a1fb9b7c2c89f5914cc670a4056bcccdd0a34/pkg/operator/resource/resourceapply/core.go#L277

but nothing injecting into pods. Where does the pod injection come from?  Is it fully novel in this code?

Reply:
As per offline discussion, named it `config.openshift.io/inject-proxy-cabundle` (to be consistent with `config.openshift.io/inject-proxy`).